src/axom/quest/MarchingCubes.hpp

```diff
@@ -288,24 +305,24 @@ class MarchingCubesSingleDomain
       return true;
 
     case MarchingCubesRuntimePolicy::omp:
-#ifdef _AXOM_MC_USE_OPENMP
+  #ifdef _AXOM_MC_USE_OPENMP
```

Comment:
Similar comment as above about run time vs. compile time selection of execution back-end...

Reply:
Changing as suggested.